client/src/app/pages/repositories/Mvn.tsx

```diff
@@ -157,26 +159,24 @@ export const RepositoriesMvn: React.FC = () => {
                   className="repo"
                   type="text"
                   aria-label="Maven Repository Size"
-                  isReadOnly
+                  aria-disabled={!isRWXSupported || isCleaning}
+                  readOnlyVariant="default"
                   size={15}
                   width={10}
                 />
                 {"  "}
-                <Button
-                  variant="link"
-                  isInline
-                  isDisabled={isCleaning}
-                  onClick={() => setIsConfirmDialogOpen(true)}
+                <ConditionalTooltip
```

Comment:
Maybe we should confirm that there is no issue changing the button type since this is how it was mocked up.

https://raw.githubusercontent.com/konveyor/enhancements/master/enhancements/tackle/tackle-git-maven-integration/images/maven-config.png

Reply:
https://github.com/konveyor/enhancements/tree/master/enhancements/tackle/tackle-git-maven-integration - Looks like we may have forgotten to add this tooltip within the same file also. Would be nice to include it as a part of this update.

Reply:
@ibolton336, yeah let's get @rromannissen to validate the style change

Reply:
@gildub Do you mind rolling that back to the original style and addressing it in a future PR? Just want to keep this rolling so we have time to test.